### PR TITLE
Issue/59 - Beautify / Enhance source code

### LINKED ISSUE #30: Complete README description
Add Installation, Usage and Developer Guide chapter as soon as the app is fully functional

### LINKED ISSUE #59: Beautify / Enhance code for STEP I
* [x] Name class variables appropriately
* [x] Set app name
* [x] Add dev environment setup section to README
* [x] Add tests to config_manager
* [x] Write doc comments for ui and main
~* [ ] Write tests for ui~
* [x] Write tests for utils
* [x] Resolve `TODO`s

### PR DESCRIPTION
**Pull request by mac641**
closes #30 
closes #59

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,56 @@
...
+paper by Edward Price, George Moore, Leo Galway and Mark Linden.
 
-soTired is a rewrite of the application presented in the [Validation of a Smartphone-Based Approach to In Situ Cognitive Fatigue Assessment](https://mhealth.jmir.org/2017/8/e125) paper by Edward Price, George Moore, Leo Galway and Mark Linden.
+## How to contribute
+PRs are very welcome! :)  
+You can set up your development environments following the steps below.
 
-## Contributions
+### App development environment setup
+1. Make sure you have flutter (version `2.5.0`) installed and `flutter doctor -v` does not have any
```
Comment (diehmlukas) [inline README.md:28]:
```suggestion
1. Make sure you have flutter (version `^2.5.0`) installed and `flutter doctor -v` does not have any 
```

File: README.md [hunk 2]
```diff
@@ -1,18 +1,56 @@
...
 
-## Contributions
+### App development environment setup
+1. Make sure you have flutter (version `2.5.0`) installed and `flutter doctor -v` does not have any 
+errors regarding Android development.
+1. Clone the repository and run `cd soTired/ && make init` to initialize git pre-commit hooks.
+1. Inside the `soTired/` directory run `make flutterPubGet` to install all depending packages.
+
+### Server development environment setup
+1. Make sure you have go (version `1.17`) installed.
```
Comment (diehmlukas) [inline README.md:34]:
```suggestion
1. Make sure you have go (version `^1.17`) installed.
```

File: README.md [hunk 3]
```diff
@@ -1,18 +1,56 @@
...
+PRs are very welcome! :)  
+You can set up your development environments following the steps below.
 
-## Contributions
+### App development environment setup
+1. Make sure you have flutter (version `2.5.0`) installed and `flutter doctor -v` does not have any 
+errors regarding Android development.
+1. Clone the repository and run `cd soTired/ && make init` to initialize git pre-commit hooks.
+1. Inside the `soTired/` directory run `make flutterPubGet` to install all depending packages.
+
```
Comment (diehmlukas) [inline README.md:32]: Perhaps we can add here that we used Android version 9.0

**Review by AnnaGsell — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (mac641) [reply README.md]: Addressed in commit 0765521

File: README.md (hunk 2 above)
Comment (mac641) [reply README.md]: Addressed in commit 0765521

File: README.md (hunk 3 above)
Comment (mac641) [reply README.md:32]: Addressed in commit 0765521

**Review by AnnaGsell — DISMISSED** [review]

**Review by carina-str — APPROVED** [review]
Looks fine for me 👍

**Review by treilik — APPROVED** [review]

**Pull request merged**